cmake/openthread_extensions.cmake

```diff
@@ -0,0 +1,121 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+set(NRFXLIB_DIR $ENV{ZEPHYR_BASE}/../nrfxlib)
+assert_exists(NRFXLIB_DIR)
+include(${NRFXLIB_DIR}/common.cmake)
+
+set(OT_WORK_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+
+# Store the configuration of the compiled OpenThread libraries
+# and set source and destination paths.
+function(openthread_libs_configuration_write)
+  unset(OPENTHREAD_SRC_DIR CACHE)
+  unset(OPENTHREAD_CONFIG_FILE CACHE)
+  unset(OPENTHREAD_LIB_FTD CACHE)
+  unset(OPENTHREAD_LIB_MTD CACHE)
+  unset(OPENTHREAD_LIB_RADIO CACHE)
+  unset(OPENTHREAD_LIB_CLI_FTD CACHE)
+  unset(OPENTHREAD_LIB_CLI_MTD CACHE)
+  unset(OPENTHREAD_LIB_NCP_FTD CACHE)
+  unset(OPENTHREAD_LIB_NCP_MTD CACHE)
+  unset(OPENTHREAD_LIB_RCP CACHE)
+  unset(OPENTHREAD_DST_DIR CACHE)
+
+  message(STATUS "OPENTHREAD_SRC_DIR=${OPENTHREAD_SRC_DIR}")
```

Comment:
Please remove, the CMake configure phase very limited in what it should print of information.
Info, such as: Toolchain, Zephyr base, board info, and similar.